compose/common.yml

```diff
@@ -187,7 +187,7 @@ services:
     healthcheck:
       retries: 2000
     ports:
-      - ${OGMIOS_PORT:-1337}:1337
+      - ${OGMIOS_PORT:-1340}:1337
```

Comment:
Yesterday you noticed this and postgres port.

Is this change a refuse to me moved in a distinct PR or did you forgot the postgres port or what else?

Thank you

Reply:
I did not forget. I changed the default port to 5435, but according to your comments that is not a good idea. I will update it.

Reply:
done: 87faa2f7068fa9426a49798a4ecd8230a311607b